src/main/resources/db/changelog/changes/10-add-categories-column.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-categories-column
+      author: Bohdan20b
+      changes:
+        - addColumn:
+            tableName: books
+            columns:
+              - column:
+                  name: categories
+                  type: varchar(255)
```

Comment:
do we need this column here? how will it work? considering that we have a related category table